shell/components/ExplorerMembers.vue

```diff
@@ -288,7 +292,7 @@ export default {
       :create-location="createLocation"
       :create-button-label="t('members.createActionLabel')"
       :is-creatable="false"
-      :type-display="t('members.clusterAndProject')"
+      :type-display="t('members.clusterMembers')"
```

Comment:
I don't think we want to make this change since in rancher clusterAndProject is more accurate.